tests/core/test_configuration.py

```diff
@@ -557,6 +557,23 @@ def test_command_from_env(self):
             # the environment variable's echo command
             self.assertEqual(test_cmdenv_conf.get('testcmdenv', 'notacommand'), 'OK')
 
+    def test_sensitive_config_values(self):
+        config_parser = AirflowConfigParser()
+        self.assertEqual(
+            config_parser.sensitive_config_values,
```

Comment:
```suggestion
            sorted(config_parser.sensitive_config_values),
```

This way at least it survives being re-ordered.